EXAMPLES.rst

```diff
@@ -1,10 +1,10 @@
 Projects using Sphinx
 =====================
 
-This is an (incomplete) alphabetic list of projects that use Sphinx or
-are experimenting with using it for their documentation.  If you like to
-be included, please mail to `the Google group
-<https://groups.google.com/forum/#!forum/sphinx-users>`_.
+This is an incomplete list of projects that use Sphinx for their documentation.
+If you would like to add a project, please create a `pull request on GitHub
+<https://github.com/sphinx-doc/sphinx/>`_ or mail to the
+`sphinx-users Google group <https://groups.google.com/forum/#!forum/sphinx-users>`_.
```

Comment:
Perhaps redirect people to a standing GH discussion or issue -- I don't subscribe to 'Sphinx-users' personally.

A